src/ot_orchestration/operators/gcs.py

```diff
@@ -51,3 +51,95 @@ def execute(self, context) -> None:
 
         blob = b.blob(self.path)
         blob.upload_from_filename(self.src)
+        self.log.info("uploaded file from %s to: %s", self.src, self.dst)
+
+
+class UploadRemoteFileOperator(BaseOperator):
+    """Custom operator that uploads a remote file to GCS.
+
+    This operator will create a GCS bucket if it does not exist and upload the
+    file from a URL to a path inside that bucket.
+
+    Args:
+        project_id: The GCP project ID. Defaults to the platform project.
+        src: The URL to the file to upload.
```

Comment:
```suggestion
        src: Source file URL
```

The name of the operator is a bit misleading.